react-client/src/App.css

```diff
@@ -1,3 +1,16 @@
 .App {
   text-align: center;
 }
+
+a {
+  text-decoration: none !important;
```

Comment:
Is there anyway to do this without using !important? !important is really bad and can lead to style inconsistencies

Reply:
I'll sort that out right away